conda_index/index/__init__.py

```diff
@@ -536,11 +542,7 @@ def index(
         Examine all changed packages under ``self.channel_root``, updating
         ``index.html`` for each subdir.
         """
-        if verbose:
-            level = logging.DEBUG
-        else:
-            level = logging.ERROR
-
+        level = verbose and logging.DEBUG or logging.ERROR
         logging_context = utils.LoggingContext(level, loggers=[__name__])
```

Comment:
I think this has been changed to do nothing, compared to the one we inherited from conda-build.